apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -1070,8 +1080,14 @@ function ErrorMessage({ error }: { error: unknown }) {
         </div>
         <div className="ltr:ml-3 rtl:mr-3">
           <p className="text-sm text-yellow-700">
-            {rescheduleUid ? t("reschedule_fail") : t("booking_fail")}{" "}
-            {error instanceof HttpError || error instanceof Error ? t(error.message) : "Unknown error"}
+            {typeof error === "string" ? (
+              error
+            ) : (
+              <>
+                {rescheduleUid ? t("reschedule_fail") : t("booking_fail")}{" "}
+                {error instanceof HttpError || error instanceof Error ? t(error.message) : "Unknown error"}
+              </>
+            )}
```

Comment:
When the time to book is over it's show this message to user. I think, the ux can be improved


![imagen](https://user-images.githubusercontent.com/525369/221301936-6863b65a-34bf-4429-a095-210dd8786290.png)

Reply:
we need a better error message even if this is quite rare

Reply:
@emrysal can you write up something?

Reply:
Can we add an action on this warning that recalls reserveSlot?

Reply:
is there any way we can prevent this error from ever happening? like smart re-routing back to the previous screen with a toast or someting? i know this error is already much more rare than the previous one

Reply:
We can increase the time automatically, if the user continues on this page.